lib/linkerd2-proxy-transport/src/source.rs

```diff
@@ -0,0 +1,42 @@
+use super::tls::PeerIdentity;
```

Comment:
Will `Endpoint` follow a similar change?

Reply:
Probably not. there are independent types for inbound and outbound endpoints and I don't think they should be combined. I'd prefer for this to go the other way... Source should just be an impl detail and not be relied on broadly.